terraform/monitoring/04_alerting_policies.tf

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Here we create alerting policies on the Service SLO for the frontend service
+# The definition of the SLO can be found in the file "03_service_slo.tf"
+# We alert on budget burn rate, alerting if burn rate exceeds twice the allotted burn rate
+# within the past hour 
+resource "google_monitoring_alert_policy" "frontend_availability_slo_alert" {
+  display_name = "Frontend Service Availability Alert Policy"
+  combiner     = "AND"
+  conditions {
+    display_name = "SLO burn rate alert for availability SLO with a threshold of 2"
+    condition_threshold {
+
+      # This filter alerts on burn rate over the past 60 minutes
+      filter     = "select_slo_burn_rate(\"projects/${var.project_id}/services/${google_monitoring_custom_service.frontend.service_id}/serviceLevelObjectives/${google_monitoring_slo.frontend_availability_slo.slo_id}\", 60m)"
+      
+      threshold_value = "2"
```

Comment:
I think you need to particularly explain how this value was determined.  Links to any Service Monitoring docs that explain the process would be good.

Reply:
I put a comment explaining why the value was chosen/what it implies and linked to some documentation from the Google SRE Workbook that explains error budget burn rates in more detail.